centreon/www/front_src/src/Header/Poller/PollerSubMenu/PollerSubMenu.tsx

```diff
@@ -0,0 +1,102 @@
+import { makeStyles } from 'tss-react/mui';
+import { isEmpty } from 'ramda';
+
+import { Typography, Button, List, ListItem } from '@mui/material';
+
+import ExportConfiguration from './ExportConfiguration';
+
+const useStyles = makeStyles()((theme) => ({
+  link: {
+    textDecoration: 'none'
+  },
+  list: {
+    padding: 0
+  },
+  listItem: {
+    '&:not(:last-child)': {
+      borderBottom: `1px solid ${theme.palette.divider}`
+    },
+    padding: theme.spacing(1)
+  },
+  pollerDetailRow: {
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  pollerDetailTitle: {
+    flexGrow: 1
+  }
+}));
+
+export interface PollerSubMenuProps {
+  allPollerLabel: string;
+  closeSubMenu: () => void;
+  exportConfig: {
+    isExportButtonEnabled: boolean;
+  };
+  issues: Array<{
+    key: string;
+    text: string;
+    total: string;
+  }>;
+  pollerConfig: {
+    isAllowed: boolean;
+    label: string;
+    redirect: () => void;
+    testId: string;
+  };
+  pollerCount: number;
+}
+
+export const PollerSubMenu = ({
+  closeSubMenu,
+  issues,
+  pollerCount,
+  allPollerLabel,
+  pollerConfig,
+  exportConfig
+}: PollerSubMenuProps): JSX.Element => {
+  const { classes, cx } = useStyles();
+
+  return (
+    <List className={classes.list} data-testid="poller-menu">
+      {!isEmpty(issues) ? (
+        issues.map(({ text, total, key }) => {
+          return (
+            <ListItem
+              className={cx(classes.listItem, classes.pollerDetailRow)}
+              key={key}
+            >
+              <Typography className={classes.pollerDetailTitle} variant="body2">
+                {text}
+              </Typography>
+              <Typography variant="body2">{total}</Typography>
+            </ListItem>
+          );
+        })
+      ) : (
+        <ListItem className={cx(classes.listItem, classes.pollerDetailRow)}>
+          <Typography variant="body2">{allPollerLabel}</Typography>
+          <Typography variant="body2">{pollerCount as number}</Typography>
+        </ListItem>
+      )}
+      {pollerConfig.isAllowed && (
+        <ListItem className={classes.listItem}>
+          <Button
```

Comment:
Btw, you can use ListItemButton: https://mui.com/material-ui/api/list-item-button/

Reply:
yes, there is also `<ListItemText>`, `<ListItemIcon` and many other etc... but none of which are perfectly suited to do what we want. 
I think the List usage in the design system is a more complex subject than just using the MUI components or not, and we could argue a lot, there is many ways to accomplish this. 
I think, if it's okay with you, that for this PR, being relatively close to the specs and using part of MUI is a good starting point. 

we should have separated discussion about our lists in the App later.
what do you think ?

Reply:
It's just a suggestion but I can make it without ListItemButton